doc/man3/OSSL_SELF_TEST_set_callback.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+OSSL_SELF_TEST_set_callback,
+OSSL_SELF_TEST_get_callback - specify a callback for processing self tests
+
+=head1 SYNOPSIS
+
+ #include <openssl/self_test.h>
+
+ void OSSL_SELF_TEST_set_callback(OPENSSL_CTX *ctx, OSSL_CALLBACK *cb);
+ OSSL_CALLBACK *OSSL_SELF_TEST_get_callback(void);
+
+=head1 DESCRIPTION
+
+OSSL_SELF_TEST_set_callback() sets the function B<cb> that is called during
+self testing that is associated with a given library context B<ctx>.
+The callback function receives information about a running self test and may
+return a result.
```

Comment:
A reference to the page which describes the callback should be given here.

Reply:
reworded